libs/sdk-backend-bear/src/convertors/fromBackend/AccessControlConverter.ts

```diff
@@ -0,0 +1,68 @@
+// (C) 2021-2023 GoodData Corporation
+import { GdcAccessControl } from "@gooddata/api-model-bear";
+
+import { convertWorkspaceUserGroup } from "./UserGroupsConverter";
+import { convertUsersItem } from "./UsersConverter";
+import {
+    AccessGranteeDetail,
+    GranteeWithGranularPermissions,
+    IAvailableUserAccessGrantee,
+    IAvailableUserGroupAccessGrantee,
+    IWorkspaceUser,
+    IWorkspaceUserGroup,
+    IAccessGrantee,
+} from "@gooddata/sdk-model";
+import isEmpty from "lodash/isEmpty";
+
+function isGranteeUserInfo(
+    grantee: GdcAccessControl.IGranteeUserInfo | GdcAccessControl.IGranteeUserGroupInfo,
+): grantee is GdcAccessControl.IGranteeUserInfo {
+    return !isEmpty(grantee) && (grantee as GdcAccessControl.IGranteeUserInfo).user !== undefined;
+}
+
+export const convertGranteeEntry = (item: GdcAccessControl.IGranteeEntry): AccessGranteeDetail => {
+    if (isGranteeUserInfo(item.aclEntry.grantee)) {
+        return {
+            type: "user",
+            user: convertUsersItem(item.aclEntry.grantee.user),
+        };
+    } else {
+        return {
+            type: "group",
+            userGroup: convertWorkspaceUserGroup(item.aclEntry.grantee.userGroup),
+        };
+    }
+};
+
+export const mapUserFullName = (user: IWorkspaceUser): string => {
```

Comment:
We should follow the fallback mechanism described in [TNT-1185](https://gooddata.atlassian.net/browse/TNT-1185) "Panther users are listed as their IDs if their name, surname, and email is not set (email is used if the user's name and surname is missing). **Same applies for Panther group if its name is missing**)."

Reply:
Particular Tiger convertor is using the fallback
https://github.com/scavnickyj/gooddata-ui-sdk/blob/3af08e4b2402351b5fdee6024048b742183c730c/libs/sdk-backend-tiger/src/convertors/fromBackend/AccessControlConverter.ts#L25-L36

Reply:
I see, OK, my bad